### PR TITLE
Reduce "panelization"

### PR DESCRIPTION
**Pull request by Emily-ejag**
Going back to having merge issues

### CODE REVIEW

File: src/pages/ScoreReport.vue [hunk 1]
```diff
@@ -73,7 +45,16 @@
             @filter="onFilter($event)"
             @export-all="exportAll"
             @export-selected="exportSelected"
-          />
+          >
+          <label for="ms-columns" class="view-label">View</label>
```
Comment (richford) [inline src/pages/ScoreReport.vue:49]: Passing in the class `view-label` works with the slot, but then the label is formatted differently than all of the other drop-down labels. Can we find a way to standardize all of the multi-select labels?

**Review by richford — CHANGES_REQUESTED** [review]

**Comment by richford** [comment]
@Emily-ejag, I just submitted a bunch of comments and suggestions in my review but neglected to write my overall review comments so I'll do it here:

- Overall, it looks really good. I like the new design.
- A lot of my comments have to do with using css variables instead of hard-coding colors in each css file. I'd prefer to use css variables so that it's easy to change them all in one place. This makes it easier to maintain and also easier to experiment with different colors during development.
- The other uses of RoarDataTable are now broken. For example, check out the "List Organizations" page at the /list-orgs route.
- I don't yet understand that cause, but when I checkout this branch, the behavior on my localhost is different than the behavior on the PR preview site. Namely it works on localhost but not on the staging site. Also, the login icons for google and clever are much bigger on the staging site. We'll have to resolve this before merging. I can co-work with you on that. But it would be helpful to resolve the other above issues first.

File: src/components/AdministratorSidebar.vue [hunk 2]
```diff
@@ -25,3 +25,35 @@ const props = defineProps({
...
+  userInfo: { type: Object, required: false, default: () => ({}) },
+  actions: { type: Array, required: false, default: () => [] },
+});
+
+const visible = ref(false);
+</script>
+
+<style scoped>
+/* Add scoped styles specific to this component */
+</style> -->
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/components/AdministratorSidebar.vue:59]: Done

File: src/assets/styles/components/button.scss [hunk 3]
```diff
@@ -3,8 +3,9 @@
   justify-content: center;
   align-items: center;
   color: var(--surface-a);
-  background: var(--primary-color);
-  border: 1px solid var(--primary-color);
+  // background: var(--primary-color);
+  background: #7F2D48;
+  border: 1px solid #7F2D48;
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/assets/styles/components/button.scss:8]: done

File: src/assets/styles/components/button.scss [hunk 4]
```diff
@@ -16,23 +17,23 @@
   border-color: var(--primary-color-hover);
 }
 .p-button:enabled:active {
-  background: var(--primary-color-hover);
+  background: #7F2D48;
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/assets/styles/components/button.scss:20]: done

File: src/assets/styles/components/input-switch.scss [hunk 5]
```diff
@@ -28,14 +28,14 @@
   background: var(--surface-400);
 }
 .p-inputswitch.p-inputswitch-checked .p-inputswitch-slider {
-  background: var(--primary-color);
+  background: #7F2D48;
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/assets/styles/components/input-switch.scss:31]: done

File: src/assets/styles/components/input-switch.scss [hunk 6]
```diff
@@ -28,14 +28,14 @@
...
 }
 .p-inputswitch.p-inputswitch-checked .p-inputswitch-slider:before {
   background: var(--surface-a);
 }
 .p-inputswitch.p-inputswitch-checked:not(.p-disabled):hover .p-inputswitch-slider {
-  background: var(--primary-color-hover);
+  background: #7F2D48;
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/assets/styles/components/input-switch.scss:37]: done

File: src/pages/ScoreReport.vue [hunk 7]
```diff
@@ -240,9 +245,10 @@ import _tail from 'lodash/tail';
 import _isEmpty from 'lodash/isEmpty';
 import { useAuthStore } from '@/store/auth';
 import { useQuery } from '@tanstack/vue-query';
-import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
 import DistributionChart from '@/components/reports/DistributionChart.vue';
-import { getSidebarActions } from '@/router/sidebarActions';
+// import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
+
+// import { getSidebarActions } from '@/router/sidebarActions';
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/pages/ScoreReport.vue]: done

File: src/pages/ScoreReport.vue [hunk 8]
```diff
@@ -1,41 +1,22 @@
 <template>
   <main class="container main">
-    <aside class="main-sidebar">
-      <AdministratorSidebar :actions="sidebarActions" />
-    </aside>
+    <!-- <aside class="main-sidebar"> -->
+      <!-- <AdministratorSidebar :actions="sidebarActions" /> -->
+    <!-- </aside> -->
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/pages/ScoreReport.vue:5]: done

File: src/pages/ScoreReport.vue [hunk 9]
```diff
@@ -1,41 +1,22 @@
...
     <section class="main-body">
-      <PvPanel :header="`Administration Score Report: ${administrationInfo?.name ?? ''}`">
-        <template #icons>
-          <button class="p-panel-header-icon p-link mr-2" @click="refresh">
-            <span :class="spinIcon"></span>
+      <!-- :header="`Administration Score Report: ${administrationInfo?.name ?? ''}`" -->
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/pages/ScoreReport.vue:7]: done

File: src/pages/ScoreReport.vue [hunk 10]
```diff
@@ -1,41 +1,22 @@
...
+      <PvPanel >
+        <!-- <template #icons>
+          <div class="w-full align-items-end">
+            <button class="p-panel-header-icon p-link mr-2 w-full" @click="refresh">
+            <span :class="spinIcon"> Refresh</span>
```
[the remark this answers is not in the document]
Comment (Emily-ejag) [reply src/pages/ScoreReport.vue:12]: done

**Comment by Emily-ejag** [comment]
The link was not longer available

**Pull request closed without merging**